packages/dd-trace/src/appsec/iast/analyzers/weak-cipher-analyzer.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+const Analyzer = require('./vulnerability-analyzer')
+
+const INSECURE_CIPHERS = [
```

Comment:
In `weak-hash-analyzer.js` we have changed it by a `Set`. We should do the same here.